src/test/java/dog/pawbook/testutil/TypicalEntities.java

```diff
@@ -73,6 +74,10 @@
     public static final Dog GENIE = new DogBuilder().withName("Genie").withSex("male")
             .withDateOfBirth("29-05-2020").withBreed("Husky").withOwnerID(13).build();
 
+    // Programs
+    public static final Program PROGRAM1 = new ProgramBuilder().withName("Basic Obedience Training")
+            .withDogs(1).withSessions("02-02-2020 18:00").withTags("puppies").build();
+
```

Comment:
this isn't used anywhere, I think it is better to leave it out and let the person who is writing program tests to everything